_scicomputing/hutchgo_overview.md

```diff
@@ -0,0 +1,82 @@
+
+The Globus service is a partnership between research labs and SciComp.  We are here to support, maintain, and inform but ultimately labs are responsible for maintaining their Globus features.
+
+#### Data Owner
+
+The data owner for a lab or project handles delegation of other roles to other lab staff (if needed).  Changes to the manager role needs to come from the owner.
+
+In most cases, the lab PI is the data owner.
+
+#### Data Managers
+
+Data Managers have nearly full control over data sharing within Globus.  These individuals will manage guest collections, request updates to configuration of the mapped collections, and will monitor and respond to communications from SciComp about management of Globus.
+
+#### Data Users
+
+If a mapped collection is marked private an additional role- the data user- is required.  Only individuals with this role (or a more privileged role) will be able to see and use a private mapped collection
+
+## Access
+
+HutchGO is available for all Hutch researchers and staff to use for data transfer needs in furtherance of the Hutch's mission. At this time HutchGO is not to be used to access or share covered (clinic) data.
+
+### Getting Set Up
+
+Getting your data set up to be accessible via Globus starts with an email to scicomp with the folowing information:
+
+ - who is the data owner (where it's not a lab PI)
+ - who are the data managers
+ - should the mapped collection be public or private
+ - which data sources should be made available via Globus- fast, scratch, economy, temp, etc.
+
+After this the data owner and managers will be given links to information about creating guest collections for collaborations and signed up for notices about the Globus services.
+
+Meanwhile SciComp will set up the Globus collection for the lab/project according to the specifications above.  SciComp will also create the groups for the various roles and assign the appropriate person(s) to those groups.  This will make data available to lab staff via their HutchNet ID.
+
+## Sharing Data
+
+Once the base mapped collection has been created, data managers will be able to create guest collections to share data internally and externally.  Once a path within a collection has been shared, those with access will be able to access the data _as you_.  Thus you must be sure that _you_ have access to the data being shared.  When sharing data there are options to configure read-write or read-only access.  The former is required for data upload, the latter will only allow download.  See [this documentation](/scicomputing/hutchgo_guest_collection) for details on creating the guest collection.
```

Comment:
This part is confusing. Maybe there should be a section clarifying on permissions on the filesystem vs globus permissions. Like what happens if I have access to a Globus collection that includes folders I don't have access to on the filesystem?

Also, i'm envisioning someone interpreting this sentence wrong: 
Once a path within a collection has been shared, those with access will be able to access the data _as you_.

So someone else (data manager) does something (create guest collections to share data) then it is shared and that person can "view all my files"?

Reply:
> Also, i'm envisioning someone interpreting this sentence wrong:

>> Once a path within a collection has been shared, those with access will be able to access the data as you.

> So someone else (data manager) does something (create guest collections to share data) then it is shared and that person can "view all my files"?

Hm.  Yeah, I can sort of see a misinterpretation. However, it's one that leads to more caution.  Basically I do want people to understand that guests are using the host's UID to access data- that's why the upstream restrictions are so important to maintain (e.g. path restrictions, read vs. read-write access, etc.)